README.md

```diff
@@ -3,41 +3,93 @@
-2. Copy your current `Team Fortress 2` installation to a new folder.
-3. Extract the ZIP download to this new folder.
-4. Make sure you don't have any configs installed.
-5. Double click `start_tf2.bat`. Note that you must have Steam running.
-6. Enjoy!
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
 
-Note: TC2 bundles the stock HUD with [CriticalFlaw's HUD fix](https://github.com/CriticalFlaw/TF2-HUD-Fix) in the custom folder
+* **TC2 is *not* the modern game, with new content/updates**
+
+You're looking for [Creators.TF](https://creators.tf/).
+
+* **TC2 is *not* an alternative, more stable game experience**
+
+TC2 is based on the 2017 TF2 source code leak, which contained an in-development version of Jungle Inferno. This build is rife with issues and incomplete features, and *is not compatible with* the modern game client.
+
+## How can I help?
+
+1. **Report bugs that exist in *both* TF2 *and* TC2 through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
+	* Bugs that are present in modern TF2 but are not present in TC2 cannot be fixed in this project.
+2. **Suggest quality of life improvements (*not balance changes*) through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
+	* These changes should not affect gameplay, and should be unanimously desired.
+3. **Install TC2 and play the game to test stability, bugfixes, and performance.**
+	* Information about multiplayer playtests can be found [on our Discord server](https://discord.gg/CuPb2zV).
+
+## Installation
+
+### Basic Install Instructions
+
+New users should follow this approach.
+
+*Note: TC2 is Windows-only (for now; see [#2](https://github.com/mastercomfig/team-comtress-2/issues/2))*
+
+1. Copy your current `Team Fortress 2` installation to a new folder.
+2. In your copied TF2 folder, delete the `tf/custom` folder, as well as `tf/cfg/autoexec.cfg`, `tf/cfg/config.cfg`, and any class configs (such as `heavyweapons.cfg` or `spy.cfg`).
+	* Do *not* delete the entire `tf/cfg` folder!
+4. [Download](https://github.com/mastercomfig/team-comtress-2/releases/latest) `game_clean.zip` from the latest release.
+5. Extract the downloaded ZIP to your copied TF2 folder, overwriting existing files.
+6. Double click `start_tf2.bat`. Note that you must have Steam running.
+7. Enjoy!
+
+### Advanced Install Instructions
+
+Contributors that want a cleaner, more 'stock' build might want to follow these instructions instead.
+*If these instructions are too confusing for you, use the basic instructions above.*
+
+1. Download a full TF2 install to a new folder using Steam's console or [DepotDownloader](https://github.com/SteamRE/DepotDownloader) (or [DepotDownloader-GUI](https://github.com/Jack-Myth/SteamDepotDownloader-GUI)).

```

Comment:
Or you could just rename then Team Fortress 2 folder in steamapps/common to something else and it would do the same if you verify files.

Reply:
A little hacky, I think this solution is a lot more flexible. I am *not* recommending it for normal users; for developers who know what they're doing, they'll want the extra flexibility this unlocks.

Reply:
Could you say what do you mean that this is more flexible? 

I think that using steam client built-in functions is way less "hacky" than using unofficial thirdparty programs.

Reply:
For the record, the very first tool mentioned is Steam itself. You can open the Steam client console and download the depots to whatever directory you want. That said, using the "verify integrity" method has one major issue - it is NOT a clean install. It will set up your Steam Cloud data in the install dir, whereas downloading depots will *not*, because it won't register them as Steam apps.

The whole point of the "advanced" install method is to create a *clean* install path, with no taint from previous installs, like lingering config files.